src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function f(a) {
+    return function f1(b) {
```

Comment:
You don't need to name functions if you return them right away.

```suggestion
  return (a) => {
    return (b) => {
```